core/src/main/java/io/questdb/cutlass/http/HttpMultipartContentListener.java

```diff
@@ -28,9 +28,7 @@
 import io.questdb.network.PeerIsSlowToReadException;
 import io.questdb.network.ServerDisconnectException;
 
-public interface HttpMultipartContentListener {
-    void onChunk(long lo, long hi) throws PeerDisconnectedException, PeerIsSlowToReadException, ServerDisconnectException;
-
+public interface HttpMultipartContentListener extends HttpContentListener {
```

Comment:
we cannot really separate words "multipart content", these two words together is a standard terminology. So is "chunk". onContent() is a made up terminology

Reply:
changed it back to `onChunk()` in https://github.com/questdb/questdb/pull/5621/commits/254cc0bf4bd03033b66f11e45a5b7c077f0883d5